src/main/java/org/acme/mongodb/servicebinding/MongoServiceBindingConverter.java

```diff
@@ -26,17 +26,17 @@
         ServiceBinding binding = matchingByType.get();
 
         Map<String, String> bindings = binding.getProperties();
-        String user = bindings.get("db.user");
-        String password = bindings.get("db.password");
-        String host = bindings.getOrDefault("db.host", "localhost");
-        String port = bindings.get("db.port");
+        String user = bindings.get("username");
+        String password = bindings.get("password");
+        String host = bindings.getOrDefault("host", "localhost");
+        String port = bindings.get("port");
         String hostPort;
         if( port == null ) {
             hostPort = host;
         } else {
             hostPort = String.format("%s:%s", host, port);
         }
-        String database = bindings.getOrDefault("db.name", "");
+        String database = bindings.getOrDefault("database", "fruit");
```

Comment:
Thanks, Jeremy! I think it would be "cheating" in terms of the demo to set this to _fruit_ here.

Reply:
fair enough, worth a shot ;) updated.